src/btree/bt_prefetch.c

```diff
@@ -25,10 +25,38 @@ __wt_btree_prefetch(WT_SESSION_IMPL *session, WT_REF *ref)
     conn = S2C(session);
     block_preload = 0;
 
-    if (!(F_ISSET(ref, WT_REF_FLAG_LEAF)) || (__wt_session_gen(session, WT_GEN_SPLIT) == 0))
-        return (WT_ERROR);
+    /* Don't add refs from trees that have eviction disabled since they are probably being closed */
```

Comment:
NP: Could we add a fullstop to the end of this comment?

Reply:
Fixed.